### DISCUSSION TITLE
Clone network from CC2531 to CC2531

### DISCUSSION THREAD

**Discussion by shajek** [Koenkk/zigbee2mqtt#6373] [Devices]
hello,
is possible to migrate from one 2531 to another 2531 (from PCB antenna to stick antena for better performance) ? Without reparing all devices :)

Thank you

**Comment by Koenkk** [comment]
Yes, just swap and start zigbee2mqtt!

**Reply by shajek** [reply]
Thank you, it is allready working and better signal, but cc2531 performance is not so good.
I guess, that migrate from 2531 to CC2538 does not work same way and plug and play, right ?

but again, thank you :)

**Reply by Koenkk** [reply]
For that you need to repair , see https://www.zigbee2mqtt.io/information/FAQ.html#what-does-and-does-not-require-repairing-of-all-devices
